index.js

```diff
@@ -5,12 +5,16 @@
 const os = require('os');
 const fs = require('fs');
 const path = require('path');
+const crypto = require('crypto');
 const execFile = require('child_process').execFile;
 const env = process.env;
 const user = env.LOGNAME || env.USER || env.LNAME || env.USERNAME;
-const configfile = '.v8flags.'+process.versions.v8+'.'+user+'.json';
 const exclusions = ['--help'];
 
+const hash = crypto.createHash('md5');
+hash.update(user);
```

Comment:
@phated, what version are you using? The default encoding works fine for me on Node 5.10.1.

Fixed the call.